CodeSniffer/Standards/Squiz/Sniffs/PHP/DiscouragedFunctionsSniff.php

```diff
@@ -41,9 +41,12 @@ class Squiz_Sniffs_PHP_DiscouragedFunctionsSniff extends Generic_Sniffs_PHP_Forb
      * @var array(string => string|null)
      */
     public $forbiddenFunctions = array(
-                                  'error_log' => null,
-                                  'print_r'   => null,
-                                  'var_dump'  => null,
+                                  'error_log'             => null,
+                                  'print_r'               => null,
+                                  'var_dump'              => null,
+                                  'var_export'            => null,
```

Comment:
@aik099 The first three were already here before I created the PR. It should help prevent having debug code in production. I don't know much about Symfony but are you going to use those components in a production environment?